contributors.js

```diff
@@ -27,4 +27,5 @@ const contributors = [
   { username: 'exilonium', message: 'The element of lie'},
   { username: 'Astro-Dude', message: 'print(Happy Hacktoberfest 2024!!)'},
   { username: '2ephir', message: 'May the Force be with you, always.' },
+  { username: 'imajij' message: 'cd hacktoberfest'}
```

Comment:
Hello @imajij , the json structure is not valid.
I approuve too quickly the PR but CICD failed for this reason. Please fix it please 😁